security/AccessControl.rst

```diff
@@ -18,11 +18,22 @@ that a user or group may perform, whether to invoke a function or access
 data, are organized into a hierarchy, and privileges may be granted at
 any level.
 
+Users and Groups can be updated using the Users and Groups Kylo admin pages:
```

Comment:
I think what I'll do is back the extra details out of this pull request and move this conversation over to the google groups forum -- it may be helpful to keep track of it there. Once we figure out how this does/should work, I'll create a separate pull request with additions to the doc.